### ISSUE TITLE
hang: waiting for bundle read lock

### ISSUE THREAD

**Issue by dckc** [endojs/endo#2163]
## Describe the bug

A dev reports that they're experimenting with changes to the contract in [dapp-offer-up](https://github.com/Agoric/dapp-offer-up). When running `yarn test` they see and endless series of

```
bundles/ waiting for bundle read lock: /Users/NNN/projects/learn/demo/contract/bundles/bundle-assetContract-js.lock in /Users/NNN/projects/learn/demo/contract/src/offer-up.contract.js
```

## Steps to reproduce

_the above is what we have_

## Expected behavior

tests should run

## Platform environment

_not reported_

## Additional context

 - https://github.com/Agoric/agoric-sdk/discussions/9118

cc @kbennett2000 @kriskowal

**Comment by dckc** [comment]
## work-around: remove bundles directory

`rm -rf bundles` made the symptoms go away

**Comment by kriskowal** [comment]
The bundle read lock does not exist on the most recent releases of `@endo/bundle-source`. This might be solved by an upgrade.

**Comment by dckc** [comment]
ok... let's track this as a dapp issue, then:

 - https://github.com/Agoric/dapp-offer-up/issues/71

**Comment by dckc** [comment]
> The bundle read lock does not exist on the most recent releases of `@endo/bundle-source`.

I'd like to cite evidence of that and close this issue. I looked in [the bundle-source changelog](https://github.com/endojs/endo/blob/master/packages/bundle-source/CHANGELOG.md) without luck.

Ah... I suppose this works:

> Replaces the lockfile mechanism with atomic renames.
-- https://github.com/endojs/endo/pull/1732

That suggests it's fixed in [v2.80 2023-09-12](https://github.com/endojs/endo/blob/master/packages/bundle-source/CHANGELOG.md#280-2023-09-12).
